Makefile

```diff
@@ -292,7 +292,7 @@ generate-ui:
 ifeq ($(USE_CONTAINER),1)
 	$(RERUN_IN_CONTAINER)
 else
-	cd ./web/ui && yarn && yarn run build
+	cd ./web/ui && yarn && yarn run build --network-timeout=1200000
```

Comment:
I don't think it's the `yarn build` that's the problem, but rather the first `yarn`, which is what downloads the dependencies.

Reply:
swing and a miss... it should be in the correct place now